data/json/items/magazine/22.json

```diff
@@ -477,5 +477,24 @@
     "ammo_type": [ "22" ],
     "flags": [ "MAG_COMPACT" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "22": 10 } } ]
+  },
+  {
+    "id": "glock44_mag",
+    "//:": "I couldn't find much conclusive data, but by all accounts it seems to share the dimensions of the G19's magazine.",
+    "looks_like": "glock17_17",
+    "type": "MAGAZINE",
+    "name": { "str": "Glock 44 10 round magazine" },
+    "description": "A compact, light-weight polymer magazine for use with the Glock 44.",
+    "weight": "70 g",
```

Comment:
Glock has the weight of the gun with and without the empty magazine, use that. The weight should be 57 grams